stable/stash/templates/cluster-role.yaml

```diff
@@ -26,21 +26,32 @@ rules:
   resources: ["*"]
   verbs: ["*"]
 - apiGroups:
-  - extensions
+  - apps
```

Comment:
Switching to apps now means the chart will only work with a 1.9 cluster. we'd like the charts to work for N and N-1 releases (1.9 and 1.8) at the least. Beta APIs are supported for 2 releases or 6 months at the moment.

Reply:
Hi @mattfarina. Thanks for reviewing the PR. I'm using the chart with 1.8 and RBAC successfully. I believe 1.8 does have the `apps` API group for deployments? It is `apps/v1beta2` in 1.8 and `apps/v1` in 1.9, but both versions have it. The `extensions` API group is actually deprecated in 1.8:
https://kubernetes.io/docs/reference/workloads-18-19/

Reply:
@mattfarina , `apps/v1beta1` was added in 1.6 https://v1-6.docs.kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
Ah, yeah... somehow I was thinking it put it to app/v1. You're right.